src/main/java/org/zalando/nakadi/controller/SubscriptionController.java

```diff
@@ -143,4 +148,35 @@ private Subscription getExistingSubscription(final SubscriptionBase subscription
                 subscriptionBase.getConsumerGroup());
     }
 
+    private final class EventTypesHolder {
```

Comment:
@adyach Haven't seen you latest comment :)
I would really prefer something like you have in you last comment. I don't think that saving few CPU cycles will help here. Creation of subscription is relatively rare operation, and this optimisation I believe will not even save 1ms, because it's just iteration of collection with few elements.